### PR TITLE
using existing polyfills for Object(.values .entries)

### PR DESCRIPTION
**Pull request by caub**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 /node-*
 /service-process.pid
 .DS_Store
+package-lock.json
```
Comment (JakeChampion) [inline .npmignore]: Please could you not ignore the package lock file, this is a very important file to keep for this project as it ensures we pull in the exact versions of third party polyfills that we have tested on BrowserStack

**Review by JakeChampion — CHANGES_REQUESTED** [review]
Need to recommit package lock file and not ignore it

**Comment by JakeChampion** [comment]
@caub I'm not sure what using third party polyfills gain us for these features as the polyfill code for them is small. Could you explain the reasoning behind this please

**Comment by caub** [comment]
Just was just a suggestion, this polyfill is the main one for Object.entries/values, by the person who made those proposals, it's well supported probably like the one you did, but it saves time and worry to rely on it probably

if you don't want it's ok, it makes sense too

I'm less and less convinced it's needed too (it's just a bit inconsistent how you added Object.values as a getter but not Object.entries

**Comment by JakeChampion** [comment]
I'm going to close this PR as we already have polyfills for these features, thanks for taking the time to create it and sorry it didn't get merged.

**Pull request closed without merging**